src/itn_uptime_analyzer/sheets.go

```diff
@@ -115,10 +115,8 @@ func (identity Identity) AppendUptime(config AppConfig, client *sheets.Service,
 
 	var nextEmptyColumn int = len(resp.Values[0])
 
-	updateRange := fmt.Sprintf("%s!%s%d", sheetTitle, string(nextEmptyColumn+65), rowIndex)
-
-	cellValue := []interface{}{strings.Join([]string{identity.uptime, "%"}, "")}
-
+	updateRange := fmt.Sprintf("%s!%s%d", sheetTitle, string(nextEmptyColumn+LETTER_A_ASCII_CODE), rowIndex)
```

Comment:
The sheet ID and column should now be configurable as well since it's that way now on the delegation backend.